ffi/src/log.rs

```diff
@@ -14,6 +14,11 @@ pub mod ffi {
     pub struct Log;
 
     impl Log {
+        /// # Panics
+        ///
+        /// If log directory creation fails.
+        ///
+        /// If tracing initialization fails.
         pub fn init_with_env() {
```

Comment:
issue: It’s actually not okay to panic here. Can you add a FIXME?

Reply:
Done. Does it look good?